custom_components/mikrotik_router/sensor.py

```diff
@@ -66,8 +79,49 @@
         ATTR_PATH: "interface",
         ATTR_ATTR: "rx-bits-per-second",
     },
+    "accounting_lan_tx": {
```

Comment:
Missing ATTR_GROUP for all accounting sensor types.
Not sure where HA would group it in integration page, can you check?